packages/channels/botpress-channel-messenger/src/incoming.js

```diff
@@ -75,7 +77,11 @@ module.exports = (bp, messenger) => {
         const mConfig = messenger.getConfig()
 
         if (mConfig.displayGetStarted && mConfig.autoResponseOption == 'autoResponseText') {
-          bp.messenger.sendText(profile.id, mConfig.autoResponseText)
+          try {
+            await bp.renderers.sendToUser(profile.id, mConfig.autoResponseTextRenderer)
```

Comment:
The code looks clean but I don't quite understand it. We're not using the `autoResponseText` anymore so how do we know what specifically to render using the `autoResponseTextRenderer`?

Reply:
@emirotin hhhm, you are right. Maybe it will be better change `mConfig.autoResponseOption == 'autoResponseText' ->  mConfig.autoResponseOption == 'autoResponseTextRenderer'`.